python/ray/tune/logger.py

```diff
@@ -103,6 +104,11 @@ def _init(self):
                 indent=2,
                 sort_keys=True,
                 cls=_SafeFallbackEncoder)
+        config_pkl = os.path.join(self.logdir, "params.pkl")
```

Comment:
if dill != None:
  ....
else:
  logger.info("Not saving pickled config since `dill` is not available.")

Reply:
Makes sense, thank you!